feat: Navbar component added

SudiptaPaul-31 opened the pull request:
# 📝 Pull Request Title

## 🛠️ Issue
- Closes #15 

## 📖 Description
Made a navbar component as per the figma design and made it responsive for all devices
- 

## ✅ Changes made
- [x] Navbar added
- [x] Added logo 
- [x] Responsive UI 

## 🖼️ Media (screenshots/videos)

https://github.com/user-attachments/assets/844c86a6-1ddd-498f-9cbf-5c5663d15a90


-

## 📜 Additional Notes
-

diegoTech14 commented on apps/frontend/tailwind.config.ts:
```diff
@@ -97,11 +97,11 @@ export default {
-        "xl": ["1.25rem", { lineHeight: "1.75rem" }],
+        xs: ["0.75rem", { lineHeight: "1rem" }],
+        sm: ["0.875rem", { lineHeight: "1.25rem" }],
+        base: ["1rem", { lineHeight: "1.5rem" }],
```
Please, keep the original font sizes

diegoTech14 commented on apps/frontend/tailwind.config.ts:
```diff
@@ -25,10 +25,10 @@ export default {
-        white: "hsl(var(--white))",
         background: "hsl(var(--background))",
         foreground: "hsl(var(--foreground))",
+        black: "#000000",
```
Please, keep the original variables, instead of change the value of the variables, create a navBlack to set the `#000000` black hex color

diegoTech14 commented:
PR looks good!



1. Could you redesign the sidebar like this?
![image](https://github.com/user-attachments/assets/69690a62-5a00-40fa-ba85-d58616fcdfe6)

2. Please, delete the white border from the navbar and sidebar

SudiptaPaul-31 commented:
Updated changes :

Removed white border below the navbar as well as the sidebar

Screenshot(s):

![image](https://github.com/user-attachments/assets/c929ac62-7a09-4a4a-b382-b2cf835dd7b6)
![image](https://github.com/user-attachments/assets/b5225a40-c187-40c3-9ba2-076678f19d48)
![image](https://github.com/user-attachments/assets/cc66bc51-75a5-4882-a179-406131065f54)

diegoTech14 commented:
Hi @SudiptaPaul-31 please, replace the color of the sidebar with the black of the navbar.

diegoTech14 commented:
@SudiptaPaul-31 Merged, good work!

Thanks for your contribution! 🚀
